src/jit/codegencommon.cpp

```diff
@@ -9740,6 +9740,33 @@ void CodeGen::genFnEpilog(BasicBlock* block)
                     }
                     break;
 
+                case IAT_RELPVALUE:
+                {
+#if defined(_TARGET_ARM_) && defined(FEATURE_FNV_MEM_OPTIMIZATIONS)
```

Comment:
The `FEATURE_FNV_MEM_OPTIMIZATION` ifdefs should be just on the VM side. They are not necessary on the JIT side since the JIT gets instructions on what to do from the VM side, and the cost of the extra checks is negligible.

Reply:
Fixed